### ISSUE TITLE
java.lang.IllegalArgumentException: Zero length string passed to TextLayout constructor

### ISSUE THREAD

**Issue by JanLuehr** [apache/jmeter#5740]
### Expected behavior

When using the Aggregated graph, "Display Graph" and the "Graph" Tag should display a graph.

### Actual behavior

UI hangs, exception is logged

```
2022-11-29 11:41:52,049 INFO o.a.j.g.u.JMeterMenuBar: setRunning(false, *local*)
2022-11-29 11:41:54,801 ERROR o.a.j.JMeter: Uncaught exception in thread Thread[AWT-EventQueue-0,6,main]
java.lang.IllegalArgumentException: Zero length string passed to TextLayout constructor.
        at java.awt.font.TextLayout.<init>(TextLayout.java:382) ~[?:?]
        at org.jCharts.chartText.TextTag.<init>(Unknown Source) ~[jcharts-0.7.5.jar:?]
        at org.jCharts.chartText.TextTag.<init>(Unknown Source) ~[jcharts-0.7.5.jar:?]
        at org.jCharts.chartText.TextTagGroup.addLabel(Unknown Source) ~[jcharts-0.7.5.jar:?]
        at org.jCharts.axisChart.AxisChart.setupAxis(Unknown Source) ~[jcharts-0.7.5.jar:?]
        at org.jCharts.axisChart.AxisChart.renderChart(Unknown Source) ~[jcharts-0.7.5.jar:?]
        at org.jCharts.Chart.render(Unknown Source) ~[jcharts-0.7.5.jar:?]
        at org.apache.jmeter.visualizers.AxisGraph.drawSample(AxisGraph.java:427) ~[ApacheJMeter_components.jar:5.5]
        at org.apache.jmeter.visualizers.AxisGraph.paintComponent(AxisGraph.java:316) ~[ApacheJMeter_components.jar:5.5]
        at javax.swing.JComponent.paint(JComponent.java:1119) ~[?:?]
        at javax.swing.JComponent.paintChildren(JComponent.java:952) ~[?:?]
        at javax.swing.JComponent.paint(JComponent.java:1128) ~[?:?]
        at javax.swing.JComponent.paintChildren(JComponent.java:952) ~[?:?]
        at javax.swing.JSplitPane.paintChildren(JSplitPane.java:1030) ~[?:?]
        at javax.swing.JComponent.paint(JComponent.java:1128) ~[?:?]
        at javax.swing.JComponent.paintToOffscreen(JComponent.java:5311) ~[?:?]
        at javax.swing.RepaintManager$PaintManager.paintDoubleBufferedImpl(RepaintManager.java:1657) ~[?:?]
        at javax.swing.RepaintManager$PaintManager.paintDoubleBuffered(RepaintManager.java:1632) ~[?:?]
        at javax.swing.RepaintManager$PaintManager.paint(RepaintManager.java:1570) ~[?:?]
        at javax.swing.BufferStrategyPaintManager.paint(BufferStrategyPaintManager.java:262) ~[?:?]
        at javax.swing.RepaintManager.paint(RepaintManager.java:1337) ~[?:?]
        at javax.swing.JComponent._paintImmediately(JComponent.java:5259) ~[?:?]
        at javax.swing.JComponent.paintImmediately(JComponent.java:5069) ~[?:?]
        at javax.swing.RepaintManager$4.run(RepaintManager.java:879) ~[?:?]
        at javax.swing.RepaintManager$4.run(RepaintManager.java:862) ~[?:?]
        at java.security.AccessController.doPrivileged(AccessController.java:399) ~[?:?]
        at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86) ~[?:?]
        at javax.swing.RepaintManager.paintDirtyRegions(RepaintManager.java:862) ~[?:?]
        at javax.swing.RepaintManager.paintDirtyRegions(RepaintManager.java:835) ~[?:?]
        at javax.swing.RepaintManager.prePaintDirtyRegions(RepaintManager.java:784) ~[?:?]
        at javax.swing.RepaintManager$ProcessingRunnable.run(RepaintManager.java:1898) ~[?:?]
        at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318) ~[?:?]
        at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:771) ~[?:?]
        at java.awt.EventQueue$4.run(EventQueue.java:722) ~[?:?]
        at java.awt.EventQueue$4.run(EventQueue.java:716) ~[?:?]
        at java.security.AccessController.doPrivileged(AccessController.java:399) ~[?:?]
        at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86) ~[?:?]
        at java.awt.EventQueue.dispatchEvent(EventQueue.java:741) ~[?:?]
        at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203) ~[?:?]
        at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124) ~[?:?]
        at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113) ~[?:?]
        at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109) ~[?:?]
        at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101) ~[?:?]
        at java.awt.EventDispatchThread.run(EventDispatchThread.java:90) ~[?:?]

```

### Steps to reproduce the problem

Execute testplan, open graph view - Testplan: https://jluehr.de/test-plan-5740.jmx

### JMeter Version

5.5

### Java Version

OpenJDK 64-Bit Server VM (build 11.0.16+8-post-Debian-1deb11u1, mixed mode, sharing)

### OS Version

Debian Linux 11

**Comment by FSchumacher** [comment]
That happens, when you have samples or controllers without a name.

I will open a PR, where those labels (as they are called in the graph) are renamed to `<empty>`.

**Comment by FSchumacher** [comment]
I have merged the PR, can you test next [nightly or build from trunk](https://jmeter.apache.org/nightly.html)?

**Comment by JanLuehr** [comment]
Great - thanks!

Fixed in apache-jmeter-5.5.1-SNAPSHOT.tgz as of 2023-01-04.
